docs/porting/third-party-deps.md

```diff
@@ -30,24 +30,29 @@ netstandard1.4
 netstandard1.5
 netstandard1.6
 netcoreapp1.0
-portable-<tfm_list>-whatever
+portable-net45-win8
+portable-win8-wpa8
+portable-net451-win81
+portable-net45-win8-wpa8-wpa81
 ```
 
-These are the Target Framework Monikers (TFM for short) which map to versions of [The .NET Standard Library](../standard/library.md).  Note that `netcoreapp1.0`, while compatible, is for applications and not libraries.  Although there's nothing wrong with using a library which is `netcoreapp1.0`-based, that library may not be intended for anything *other* than consumption by other `netcoreapp1.0` applications.
+These are the Target Framework Monikers (TFM for short) which map to versions of [The .NET Standard Library](../standard/library.md) and traditional Portable Class Library (PCL) profiles which are compatible with .NET Core.  Note that `netcoreapp1.0`, while compatible, is for applications and not libraries.  Although there's nothing wrong with using a library which is `netcoreapp1.0`-based, that library may not be intended for anything *other* than consumption by other `netcoreapp1.0` applications.
 
 There are also some legacy TFMs used in pre-release versions of .NET Core that may also be compatible:
 
 ```
 dnxcore50
-dotnet1.0
-dotnet1.1
-dotnet1.2
-dotnet1.3
-dotnet1.4
-dotnet1.5
+dotnet5.0
+dotnet5.1
+dotnet5.2
+dotnet5.3
+dotnet5.4
+dotnet5.5
 ```
 
-**While these may work with your code, there is no guarantee of compatibility**.  Be on the lookout for when these packages update to The .NET STandard Library.
+**While these will likely work with your code, there is no guarantee of compatibility**.  Packages with these TFMs were built with pre-release .NET Core packages.  Take note of when (or if) packages like this are updated to be `netstandard`-based.
+
+> **Note:** To use a package targeting a traditional PCL or pre-release .NET Core target, you must use the `imports` directive in your `project.json` file.
```

Comment:
nit: notes on ops should have the following syntax
```
> [!NOTE]
> This is NOTE
```

I'll be changing them next week if you don't wanna change it now